source/user.scripts/pkg_build.sh

```diff
@@ -4,10 +4,23 @@ tmpdir=/tmp/tmp.$(( $RANDOM * 19318203981230 + 40 ))
 plugin=$(basename ${DIR})
 archive="$(dirname $(dirname ${DIR}))/archive"
 version=$(date +"%Y.%m.%d")$1
+ACE_VERSION=1.4.14
 
 mkdir -p $tmpdir
 
 cp --parents -f $(find . -type f ! \( -iname "pkg_build.sh" -o -iname "sftp-config.json"  \) ) $tmpdir/
+
+#Install Ace Editor
+mkdir -p $tmpdir/usr/local/emhttp/plugins/user.scripts/javascript/ace/
+wget --no-check-certificate https://github.com/ajaxorg/ace-builds/archive/refs/tags/v${ACE_VERSION}.zip
+mkdir -p /tmp/ace
+unzip v${ACE_VERSION}.zip "ace-builds-${ACE_VERSION}/src-min-noconflict/*" -d "/tmp/ace"
```

Comment:
In compose.manager i only included the ace file that i specifically needed. Here i went with the bonienl's approach of including all of ace.